### PR TITLE
Community Article - Connect to Cloud Servers with Filezilla via sFTP

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bd924ba0fe/how-to/)

**Comment by stephamon** [comment]
Hi @cyrichardson can you edit this one as well?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1b6d7ef081/how-to/)

**Comment by cyrichardson** [comment]
@stephamon On it!

**Comment by cyrichardson** [comment]
@stephamon Some suggestions here: 
[connect-to-cloud-servers-with-filezilla-by-using-sftp_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2797626/connect-to-cloud-servers-with-filezilla-by-using-sftp_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f38f02aba8/how-to/).

File: content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+permalink: connect-to-cloud-servers-with-filezilla-by-using-sftp
```
Comment (RackerKate) [inline content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md]: I think you need to add a forward slash at the end of the permalink.

File: content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Like Secure Shell (SSH), SFTP runs on port 22 by default on cloud servers. If you have changed
+the port that SSH listens on, then you must use that new port for SFTP as well.
+
+You can connect to your server by using SFTP with the root user but we recommended creating a new
+system user for security purposes. For more information, see
+[Basic Cloud Server security](/how-to/basic-cloud-server-security).
+
+### Configure Filezilla
+
+Download Filezilla from https://filezilla-project.org/ and then install it. You need the
```
Comment (RackerKate) [inline content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md]: Suggest either hyperlinking the URL or using the URL to hyperlink the word "Filezilla."

File: content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+following information during the configuration process:
+
+- Hostname: The public Internet Protocol (IP) address of your cloud server.
```
Comment (RackerKate) [inline content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md]: Not a big deal but maybe make the field names bold?

File: content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- User: We recommend using a new system user rather than using root. However, the root user works and we are using root for this article.
+- Password: The password for the user you are logging in with.
+
+Use the following steps to configure Filezilla to use SFTP:
+
+1. Launch Filezilla and input the following information:
+   
+   - Host: Your servers public IP address
```
Comment (RackerKate) [inline content/cloud-servers/connect-to-cloud-servers-with-filezilla-by-using-sftp.md]: Bold field names?

**Pull request merged**